boot/initramfs20dirs.go

```diff
@@ -50,6 +54,7 @@ var (
 func setInitramfsDirVars(rootdir string) {
 	InitramfsRunMntDir = filepath.Join(rootdir, "run/mnt")
 	InitramfsUbuntuDataDir = filepath.Join(InitramfsRunMntDir, "ubuntu-data")
+	InitramfsRecoverUbuntuDataDir = filepath.Join(InitramfsRunMntDir, "recover-ubuntu-data")
```

Comment:
not 100% happy with this name, in hindsight maybe the ephemeral ubuntu-data shouldn't be called ubuntu-data, the issue it's that it is hardcoded in the initramfs writable handling, so having a different values for ephemeral vs run mode is annoying for that :/